CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 Project Changelog
 =================
 
+Release 1.2.2
+-------------
+* Added a submodule for plotting, deprecating SOLPSMesh methods
```

Comment:
if you look into the change log of the core package, you can see we add the issue number at the end of the line. I think in general this should be followed whenever possible. Please add the issue number.